src/Artsy/Router/buildServerApp.tsx

```diff
@@ -26,19 +27,19 @@ interface Resolve {
 // No need to invoke this for each request.
 const MediaStyle = createMediaStyle()
 
-export function buildServerApp(config: RouterConfig): Promise<Resolve> {
+export function buildServerApp(
+  config: RouterConfig & { userAgent: string | undefined }
```

Comment:
Why not add `userAgent` to `RouterConfig`?

Reply:
I figure that would make it required by the client app too, which isn’t what we’d want.